presto-cli/src/test/java/com/facebook/presto/cli/TestTableNameCompleter.java

```diff
@@ -27,7 +27,6 @@ public void testAutoCompleteWithoutSchema()
     {
         ClientSession session = new ClientOptions().toClientSession();
         QueryRunner runner = createQueryRunner(session);
-
```

Comment:
unrelated change

Reply:
sorry, was messed up when cherrying pick from the other branch. Fixed